src/StorageKey.elm

```diff
@@ -65,10 +60,5 @@ isNew storageKey =
 
 -}
 isExisting : StorageKey recordId -> Bool
-isExisting storageKey =
-    case storageKey of
-        Existing _ ->
-            True
-
-        _ ->
-            False
+isExisting =
+    not << isNew
```

Comment:
nice! can I usually prefer to avoid implicit arguments (unless it's a very simple function).

How about:

```elm
isExisting storageKey = 
     not <| isNew storageKey
```